phpBB/adm/style/admin.css

```diff
@@ -1126,6 +1126,24 @@ input, textarea {
 	border-bottom: 1px solid #D5D5C8;
 }
 
+.rtl input[type="email"], .rtl input[type="url"],
+.rtl #add_extension, .rtl #ban,
```

Comment:
Why? just add the utility class to the html! We do not target IDs for css as its a bad practice and leads to over specification

Reply:
The previous solution included a special class that was added in a rather weird way by extending the type specifier of acp options. Adding a CSS class that takes care of this was not what bothered me, only the way it was added. Sorry for the confusion.

Reply:
Editing `build_cfg_template` function is required, An alternate way is adding new non HTML types for ACP options: `path` because most of them are paths, `textltr` for others.

And editing  `build_cfg_template` function to change these new types to `type="text" class="always-ltr"`

Reply:
If it’s doable via ids then it’s doable via two class approach! Just have to factor out the specificity.

Reply:
Most likely I would ha e to look into it further to see what exactly you are trying to accomplish